src/app/pages/quickMaps/components/download/download.component.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
Small file, doesn't look like it should need any of these.

Reply:
removed